x-pack/legacy/plugins/siem/public/components/open_timeline/timelines_table/icon_header_columns.test.tsx

```diff
@@ -21,32 +21,34 @@ jest.mock('../../../lib/kibana');
 describe('#getActionsColumns', () => {
   const theme = () => ({ eui: euiDarkVars, darkMode: true });
   let mockResults: OpenTimelineResult[];
-
+  const getDefaultProps = (mockOpenTimelineResults: OpenTimelineResult[]): TimelinesTableProps => ({
```

Comment:
it looks that you are re-using this getDefaultProps more than one time, I think that you can create a mock file so every test is using the same.